docker-images/kafka/Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG THIRD_PARTY_LIBS
 ARG strimzi_version
 ARG TARGETPLATFORM
 
-RUN microdnf install gettext nmap-ncat stunnel net-tools unzip hostname findutils tar \
+RUN microdnf install gettext nmap-ncat stunnel net-tools unzip hostname findutils tar maven \
```

Comment:
I don't think we want this for security reason. As discussed before offline - I think this should use multistage build or some other way to download it without having Maven in all the container images.

Reply:
@sknot-rh have you seen https://github.com/shrinkwrap/resolver?